docs/using-scylla/index.rst

```diff
@@ -5,47 +5,40 @@ ScyllaDB for Developers
    :maxdepth: 2
    :hidden: 
 
-   learn
-   Scylla Alternator <alternator/index>
-   Scylla Features <features>
-   Scylla Drivers <drivers/index>
+   Develop with ScyllaDB <https://docs.scylladb.com/stable/get-started/develop-with-scylladb/index.html>
+   Tutorials and Example Projects <https://docs.scylladb.com/stable/get-started/develop-with-scylladb/tutorials-example-projects.html>
```

Comment:
Why are these links external instead of relative? Are we building the "get-started" directory from pieces from scylla.git and other pieces from other sources? Because I don't see a "develop-with-scylladb" in scylla.git's docs/getting-started.